django_rq/cron_views.py

```
@@ -22,27 +55,91 @@ def cron_scheduler_detail(request, connection_index: int, scheduler_name: str):
+        connection, serializer = queue.connection, queue.serializer
+    else:
+        connection, serializer = scheduler.connection, None
+
+    num_jobs = get_cron_job_history_count(cron_job, scheduler.connection)
+    page = int(request.GET.get('page', 1))
+
+    history: list[dict[str, Any]] = []
+    page_range: list[int] = []
+
+    if num_jobs > 0:
+        last_page = int(ceil(num_jobs / ITEMS_PER_PAGE))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)